docker-compose.yml

```diff
@@ -335,6 +335,47 @@ services:
     profiles:
       - elasticsearch
 
+  comms:
+    build:
+      context: comms
+      dockerfile: DockerfileM1
+    container_name: comms
+    command: /comms-linux discovery
+    environment:
+      audius_db_url: "postgres://postgres:postgres@discovery-provider-db:5432/postgres?sslmode=disable"
```

Comment:
Comms could either re-use the discovery postgres... or run it's own as is done here.

If it does do it's own we'd have to seed the schema from the schema dump tho.